.github/workflows/cd.workers-new.yml

```diff
@@ -0,0 +1,31 @@
+name: (CD) workers.new
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'packages/workers.new/**'
+
+jobs:
+  deploy:
+    name: Node.js v16
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/create-cloudflare
```

Comment:
```suggestion
        working-directory: packages/workers.new
```

Reply:
should this run in workers.new instead?

Reply:
yup thanks, i see the file is unsaved locally